webviz_ert/controllers/response_correlation_controller.py

```diff
@@ -489,7 +489,7 @@ def update_corr_param_resp(
         responses: List[str],
         parameters: List[str],
         corr_param_resp: Dict,
-    ) -> Optional[Dict]:
+    ) -> Dict:
```

Comment:
Do you understand why before we had Optional?

Reply:
i tried to google it in the context of dash callbacks, and found nothing.

in general i guess it's for when you want to have a `return None` in your code sometimes. but we don't haz that. I thought it was connected to the `raise PreventUpdate` that we had here or there, but I suppose in that case we are raising an exception, and not returning control flow in the usual manner, so the typing of the return shouldn't apply!? i didn't get too deep into the latter point.

in any way, i think it's just not necessary, as we always have a return value, and it's type is always `Dict`. And maybe we also want there to always be a dictionary returned?

does that make sense?

Reply:
yes, ok! In any case, if it was necessary we will find out very soon :D